### DISCUSSION TITLE
Welcome to vue-cursor-fx Discussions!

### DISCUSSION THREAD

**Discussion by LuXDAmore** [LuXDAmore/vue-cursor-fx#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pooreffects** [comment]
Howdy! I have integrated cursor fx in Nuxt as a component yet the configuration doesn't seem to work except for the cursor color!
<cursor-fx 
color="#534875"
color-hover="#fff"
.....
>
Where should I set up the configuration for the on hover state and other things!?
Thanks a million

**Reply by LuXDAmore** [reply]
Ehy @Pooreffects !

Can you provide a reproduction link? Maybe on [CODESANDBOX](https://codesandbox.io/) so it's easy to check if something is wrong!

Thank you very much!  😄

**Comment by Nicolaseliell** [comment]
Hi! First of all apologies for my english! 

I installed cursor fx to my nuxt project and I'm pretty new on nuxt and integrating modules. My problem is that I don't know how to configure the cursor to the styles I want hover color, size, animation etc. Where can I configure this stuff or is it possible? And is it possible to leave the default cursor visible while using the plugin?

Thank you in advance!

**Comment by SYBzone** [comment]
Hey, i ve started to use the cursor and it seems working correctly, i havent started to custom it bc i actualy have an iframe for a spline element, like 3D and interactive, and i dont suceed pass the cursor on it, it s staying outside the element and coming back to the default cursor when i pass on it.  Somebody have an idea  ?

**Reply by mansurkhoja** [reply]
Cursor not work in iframe
